### ISSUE TITLE
Makes zoom by clicks and taps actions

### ISSUE THREAD

**Issue by yanmagalhaes-ir** [GoogleChromeLabs/pinch-zoom#2]
How do you think about a feature that makes a zoom in a pinch-zoom component, in accord to click and taps user actions? 

Its an alternative and direct option, instead of a pinch, specially at mobile devices.

**Comment by jakearchibald** [comment]
Are you talking about double-tap to zoom?

**Comment by yanmagalhaes-ir** [comment]
Exactly. A one tap to zoom-in and double tap to zoom out. Do you like it?

**Comment by jakearchibald** [comment]
Not really. It means that any content inside the pinch-zoom loses its ability to be interactive.

Double-tap may be possible, as it's a much less common interaction.

**Comment by DanielSokil** [comment]
Double tap should be used to reset element zoom.

**Comment by ghost** [comment]
As an example, google maps uses a double click/tap to zoom in to the point clicked/tapped.
